common/src/main/java/net/vyhub/config/VyHubConfiguration.java

```diff
@@ -77,6 +78,8 @@ public static String getAdvertPrefix() {
 
     public static Boolean getIsBackendServer() { return Boolean.valueOf(config.get("is_backend_server")); }
 
+    public static Boolean getSuccesMessage() { return Boolean.valueOf(config.get("send_succes_annoucments")); }
```

Comment:
This will probably throw an exception for all existing gameservers, as the config-key does not exist yet. I propose providing a default, when the key is not yet part of the config.

Reply:
i at the time don't know how to fix this but the other things are fixed